auth-web/tests/unit/services/business-lookup.service.spec.ts

```diff
@@ -14,15 +14,14 @@ describe('Business Lookup Services', () => {
     }
 
     const url = BusinessLookupServices.registriesSearchApiUrl + 'businesses/search/facets?start=0&rows=20' +
-      '&categories=legalType:A,BC,BEN,C,CC,CP,CUL,FI,GP,LL,LLC,LP,PA,S,SP,ULC,XCP,XL,XP,XS' +
-      `&query=value:${encodeURIComponent('FM1000002')}`
+      `&categories=legalType:SP&query=value:${encodeURIComponent('FM1000002')}`
```

Comment:
Instead of updating this list, which would serve no purpose here, I simplified this.